release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3209,3 +3209,19 @@ $ oc adm release info 4.12.3 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+// TODO: Include an update like this in whichever 4.12 z-stream this gets backported to:
+[id="ocp-4-12-x-features"]
+==== Features
+
+[id="ocp-4-12-x-psa-tp-feature-set"]
+===== Pod security admission restricted enforcement (Technology Preview)
+
+With this update, pod security admission restricted enforcement is available as a Technology Preview feature by enabling the `TechPreviewNoUpgrade` feature set. Now, if you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
+
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
```

Comment:
it prevents any upgrades, doesn't it?

Reply:
@stlaz This warning wording is taken straight from the feature sets documentation, so would like to keep the same wording:

https://docs.openshift.com/container-platform/4.12/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-about_nodes-cluster-enabling

Reply:
:+1:

Reply:
I think @stlaz might be right: https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L32-L33

Who might be able to confirm? 

@bergerhoffer This can be addressed in a separate PR, if needed, so we can CP back, if needed